parsec/test_utils/organization.py

```diff
@@ -116,16 +111,23 @@ async def initialize_test_organization(
                 # Add additional random users
                 if additional_users_number > 0:
```

Comment:
we can remove the if check and always call `_add_random_users` given it does nothing but a loop on `range(additional_users_number)`